bot/resources/tags/ot.md

```diff
@@ -1,3 +1,9 @@
-**Off-topic channel:** <#463035268514185226>
+---
+embed:
+    title: "Off-topic channel"
+---
+<#291284109232308226>
+<#463035241142026251>
+<#463035268514185226>
```

Comment:
Only one channel was mentioned in this tag for a reason, could you please change it back.